central/graphql/resolvers/cluster_vulnerabilities.go

```diff
@@ -0,0 +1,90 @@
+package resolvers
+
+import (
+	"context"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/central/metrics"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/features"
+	pkgMetrics "github.com/stackrox/rox/pkg/metrics"
+	"github.com/stackrox/rox/pkg/search"
+	"github.com/stackrox/rox/pkg/utils"
+)
+
+func init() {
+	schema := getBuilder()
+	utils.Must(
+		schema.AddType("ClusterVulnerability",
+			append(commonVulnerabilitySubResolvers,
+				"vulnerabilityType: String!",
```

Comment:
Just a note that we can enforce non-nil only because the default enum string value is `UNKNOWN_VALUE`.